specs/workspace_spec.rb

```diff
@@ -0,0 +1,106 @@
+require_relative "test_helper.rb"
+
```

Comment:
In this file you also need to have some negative-case tests with invalid arguments.